wpilibj/src/main/java/edu/wpi/first/wpilibj/smartdashboard/MechanismRoot2d.java

```diff
@@ -36,42 +31,20 @@ public final class MechanismRoot2d implements AutoCloseable {
    * @param y y coordinate of root (provide only when constructing a root node)
    */
   MechanismRoot2d(String name, double x, double y) {
-    m_name = name;
+    super(name);
     m_x = x;
     m_y = y;
   }
 
   @Override
   public void close() {
+    super.close();
```

Comment:
I think this should be at the end of the method and not at the start -- like popping a stack.